site/_posts/2021-04-13-oggeh-fusionauth-gluu.md

```diff
@@ -0,0 +1,92 @@
+
+> Literarily 40% of our expenses [are] just gone [after adopting FusionAuth]. 
+
+**Dan:** Do your customers have the ability to customize the themes and email templates or is that something you take care of for them?
+
+**Ahmed:** That depends, in most cases, we do it ourselves for non-technical customers. We plan to extend our API services for those who have a technical background and the ability to use our platform tools (i.e. frontend developers). So that they can inject those themselves.
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Ahmed:** Gluu requires 2xCPU and 8G Memory to start and operate properly. I find that remarkably unnecessary even if we have thousands of users. Additionally, there's no notion of having multiple tenants in Gluu. Same for themes and email templates. That wasn't an issue as long as we used it to authenticate our own users only, but later, we had to add user authentication to other applications (especially e-commerce). It was almost a nightmare. Replicating the whole server was the only option!
+
+There's also one important note I must add here. Gluu might break due to expired JKS files for SCIM. These Java Keystore files are generated upon installation and [expire after one year](https://gluu.org/docs/gluu-server/4.0/operation/replace-expired-jks-scim/). Imagine how we felt after exactly one year of the installation to find a broken Gluu and spending many hours digging up this annoying fact!
+
+To be honest, that was the moment where I decided to get rid of Gluu. There were many other alternatives. FusionAuth caught my attention and just stuck there. I used to say that Auth0 has the best documentation and comprehensive service. But all that changed dramatically after I explored FusionAuth. It works like a charm from the first attempt. That's something that doesn't usually happen when trying new things. I wish to thank you for that.
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Ahmed:** A tremendous amount! Literarily 40% of our expenses just gone. No more unnecessary high resources for Gluu. FusionAuth works just fine on our auto-scaled stack.
+
+> [FusionAuth's] footprint is remarkably small and does not require great CPU or memory resources to operate.
+
+**Dan:** How do you run FusionAuth (k8s, standalone tomcat server, behind a proxy, etc)?
+
+**Ahmed:** K8s. We do the same for the rest of OGGEH Cloud Platform tools, which includes a variety of auto-scaled micro-services. Now FusionAuth is one of those. I did not exactly pick the docker version provided by FusionAuth docs. I had to modify the official Dockerfile from FusionAuth/fusionauth-containers repository for auto-installing a custom plugin. One that handles LDAP Salted SHA512 passwords (imported from Gluu). 
```

Comment:
wonder if this is a bit of compromising information or if it is inconsequential `LDAP Salted SHA512 passwords`

Reply:
Hmmm. He didn't seemed concerned about sharing it. Maybe he is rehashing the passwords?